api/v1/middleware/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
 	auth: require('./auth.js'),
 	errors: require('./errors.js'),
 	permission: require('./permission.js'),
-	request: require('./request.js'),
 	response: require('./response.js'),
+	request: require('./request.js'),
```

Comment:
Why is the request require now done after response, is there an explicit intent to it?